include/pmacc/PMaccConfig.cmake

```diff
@@ -141,6 +141,11 @@ if((NOT ALPAKA_ACC_GPU_CUDA_ENABLE) AND ALPAKA_ACC_GPU_CUDA_ONLY_MODE)
     message(WARNING "ALPAKA_ACC_GPU_CUDA_ONLY_MODE is set to OFF because cuda backend is not activated")
 endif()
 
+# includes and definitions are already added in cupla's & alpaka's
```

Comment:
`alpaka_add_executable` does not add any definitions or includes. This is all done by cupla.
I do not intend to add those within alpaka because `alpaka_add_executable` should be identical to the CMake `add_executable` in which it only creates a target with the given files.